classes/AutoPanMix.sc

```diff
@@ -323,7 +327,7 @@ AutoPanMix : BaseMix {
 						// create a unique mix for jamulus that excludes itself
 						mix[0] = 0;
 					}, {
-						mix[clientNum] = selfVolume;
+						mix[clientNum] = selfVolume * selfVolumeCalibration;
```

Comment:
i don't think it's correct to change the meaning of 100%. currently this means i will hear myself at the same level that everyone else hears me.